node.gyp

```diff
@@ -186,8 +186,7 @@
       'include_dirs': [
         'src',
         'tools/msvs/genfiles',
-        '<(SHARED_INTERMEDIATE_DIR)', # for node_natives.h
-        'deps/nghttp2/lib/includes'
+        '<(SHARED_INTERMEDIATE_DIR)' # for node_natives.h
```

Comment:
Can you keep the comma?  Makes the diff less noisy (also when a line is added later.)